### PR TITLE
id3 data

### PR DESCRIPTION
**Pull request by Saurabh7**

### CODE REVIEW

File: testsuite/meta/classifier/id3.dat [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<<_SHOGUN_SERIALIZABLE_ASCII_FILE_V_00_>>
+array Vector<SGSerializable*> 2 ({WrappedBasic float64 [
+value float64 0.1
+]}{WrappedSGVector float64 [
+value SGVector<float64> 20 ({2}{2}{2}{2}{2}{2}{2}{2}{2}{2}{2}{2}{2}{2}{2}{2}{2}{2}{2}{2})
```
Comment (karlnapf) [inline testsuite/meta/classifier/id3.dat]: this seems a fishy result

**Comment by karlnapf** [comment]
Any idea why the result looks like this?

**Comment by Saurabh7** [comment]
Not sure, will try with different data and check.

**Comment by iglesias** [comment]
Any updates, @Saurabh7?

**Pull request closed without merging**